tests/basic/pure_test.go

```diff
@@ -4143,3 +4143,129 @@ var _ = Describe("{CreateNewPoolsWhenFadaFbdaVolumeCreationInProgress}", func()
 	})
 
 })
+
+var _ = Describe("{TestParallelPxAndFadaVolumeResize}", func() {
+	/*
+		https://purestorage.atlassian.net/browse/PTX-23985
+		1. Deploy an app which uses FADA volume( uses csi provisioner and a pure_block backend) and an app which uses base volume(uses portworx-volume as provisioner)
+		2. Parallely resize the pvc of FADA deployed app and also resize the base app pvc
+		3. Validate the volume resize
+	*/
+	JustBeforeEach(func() {
+		StartTorpedoTest("TestParallelPxAndFadaVolumeResize", "Px Volume Resize in parallel to FADA/FBDA Volume Resize ( PVC Resize )", nil, 0)
+	})
+	var contexts []*scheduler.Context
+	itLog := "Px Volume Resize in parallel to FADA/FBDA Volume Resize ( PVC Resize )"
+	It(itLog, func() {
+		log.InfoD(itLog)
+		var resizedVols []*volume.Volume
+
+		stepLog := "Deploy applications"
+		Step(stepLog, func() {
+
+			Inst().AppList = []string{"fio-cloudsnap"}
```

Comment:
why hardcoding apps ?, cant we pass the apps from job and pick

Reply:
so i thought it would be better we send it hardcoded , as our test case needs one normal app deployment and one fada app deployment